pyvrp/GeneticAlgorithm.py

```diff
@@ -163,49 +158,22 @@ def add_and_register(sol):
             self._pm.register_load_feasible(not sol.has_excess_load())
             self._pm.register_time_feasible(not sol.has_time_warp())
 
-        intensify_prob = self._params.intensify_probability
-        should_intensify = self._rng.rand() < intensify_prob
-
-        sol = self._ls.run(sol, self._cost_evaluator, should_intensify)
+        sol = self._ls.run(sol, self._cost_evaluator)
+        add_and_register(sol)
 
         if is_new_best(sol):
             self._best = sol
 
-            # Only intensify feasible, new best solutions. See also the repair
-            # step below. TODO Refactor to on_best callback (see issue #111)
```

Comment:
Should we also close #111?

Reply:
Maybe; I'm not sure. I think we can keep that one around a little longer because I think the idea of registering custom code here and there is useful. But it might not be in the form of #111.